runner.go

```diff
@@ -1 +1,63 @@
 package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/CodeCollaborate/Server/modules/config"
+	"github.com/CodeCollaborate/Server/modules/handlers"
+	"github.com/CodeCollaborate/Server/modules/rabbitmq"
+	"github.com/CodeCollaborate/Server/utils"
+)
+
+var addr = flag.String("addr", "0.0.0.0:80", "http service address")
+
+func main() {
+
+	flag.Parse()
+	log.SetFlags(0)
+
+	// START DB CONNECTION
+	//managers.ConnectMGo()
+	//defer managers.GetPrimaryMGoSession().Close()
+
+	err := config.InitConfig()
+	if err != nil {
+		log.Fatal(err)
+	}
+	config := config.GetConfig()
+
+	// Get working directory
+	dir, err := os.Getwd()
+	if err != nil {
+		fmt.Printf("Fatal error: Could not get Working Directory: %s\n", err)
+		log.Fatal(err)
+	}
+	fmt.Println("Running in directory: " + dir)
+
+	AMQPControl := utils.NewControl()
+
+	rabbitmq.SetupRabbitExchange(
```

Comment:
Because lines 27 through 39 aren't described explicitly and aren't self-explanatory, it make this portion of the code (43-54) difficult to understand without digging into the config modules.